http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BaseRuntimeInterpreter.scala

```diff
@@ -126,7 +126,8 @@ abstract private[util] class BaseRuntimeInterpreter[F[_]: Async: ContextShift: L
           .toRight(new Exception(s"Unable to update welder because current welder image is not available"))
           .flatMap(x =>
             x.registry match {
-              case Some(ContainerRegistry.GCR)       => Right(config.imageConfig.welderGcrImage.imageUrl)
+              case Some(ContainerRegistry.GCR) | Some(ContainerRegistry.GHCR) =>
```

Comment:
is this for potentially using welder in ghcr in the future?

Reply:
Don't think we have a reason to - was just making the match exhaustive. So a Dockerhub container will cause us to use Dockerhub welder, else we'll use the GCR welder.